CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.1]
+### Added
+- Add additional tests to cover encoded data values that were not previously covered which led to unintended changes to encoding hexstrings in `v5.0.0` ([#274](https://github.com/MetaMask/eth-sig-util/pull/274))
+
```

Comment:
Nit: Not sure this is informative for users of this package. The functional change is covered already below.
```suggestion
```